### PR TITLE
add wannnier90

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/wannier90/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cp ${RECIPE_DIR}/make.inc ${SRC_DIR}/make.inc
+make wannier
+make test-serial
```
Comment (chrisburr) [inline recipes/wannier90/build.sh:4]:
```suggestion
make wannier -j$CPU_COUNT
make test-serial -j$CPU_COUNT
```

File: recipes/wannier90/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - {{ compiler('fortran') }}
+    - make
+  host:
+    - libblas
+    - liblapack
+  run:
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/wannier90.x
```
Comment (chrisburr) [inline recipes/wannier90/meta.yaml:27]: Can you try executing either of these in any way? or is that too complex?

Comment (jan-janssen) [reply recipes/wannier90/meta.yaml:27]: I thought that is fine if I use `make test-serial` - meaning I test the executable after the compilation.

Comment (chrisburr) [reply recipes/wannier90/meta.yaml:27]: It's good to know if the package survives being relocated and that all of the necessary run requirements are in place

Comment (chrisburr) [reply recipes/wannier90/meta.yaml:27]: Though of course, if running a command takes a long time or it requires special input data then it's not necessarily worth the hassle. It's up to you.

Comment (jan-janssen) [reply recipes/wannier90/meta.yaml:27]: I can try to add the input test files and then execute the test after the copy. But I would prefer to do this once the recipe is merged. I also want to try building a Version for Mac Os X then.

**Pull request merged**